spec/indico_batch_spec.rb

```diff
@@ -21,6 +21,15 @@
     expect(Set.new(response[1].keys)).to eql(expected_keys)
   end
 
+  it 'should tag text with correct emotion tags' do
```

Comment:
We test all of that on the server -- all we would really be testing is if the keyword arguments are passed along, which is independent of any particular API